fairseq/models/lstm.py

```diff
@@ -197,6 +199,9 @@ def _forward(self, input_tokens, encoder_out):
         attn_scores = attn_scores.transpose(0, 2)
 
         # project back to size of vocabulary
+        if hasattr(self, 'additional_fc'):
+            x = self.additional_fc(x)
+            x = F.dropout(x, p=self.dropout, training=self.training)
```

Comment:
I think we want `p=self.dropout_out` here.

Reply:
Sorry, I forgot I've changed to one dropout in my own code.